src/ccnl-core/src/ccnl-pkt-util.c

```diff
@@ -220,3 +220,23 @@ ccnl_cmp2int(unsigned char *cmp, size_t cmplen)
 
     return 0;
 }
+
+uint64_t
+ccnl_pkt_interest_lifetime(const struct ccnl_pkt_s *pkt)
+{
+    switch(pkt->suite) {
+#ifdef USE_SUITE_CCNTLV
+    case CCNL_SUITE_CCNTLV:
+        /* CCN-TLV parser does not support lifetime parsing, yet. */
+        return CCNL_INTEREST_TIMEOUT;
```

Comment:
I probably would add a note that the constant is in seconds while the NDN timeout value is actually in milliseconds (hence the ``uint64_t`` return type).